esp/src/eclwatch/nls/hpcc.js

```diff
@@ -132,13 +132,15 @@ define({root:
     Definition: "Definition",
     DefinitionID: "Definition ID",
     Definitions: "Definitions",
+    DefinitionDeleted: "Definition deleted",
     Delete: "Delete",
     DeleteBinding: "Delete Binding",
     DeletedBinding: "Deleted Binding",
     Deleted: "Deleted",
     DeleteEmptyDirectories: "Delete empty directories?",
     DeleteDirectories: "Remove empty directories. Do you wish to continue?",
     DeletePrevious: "Delete Previous",
+    DeleteSelectedDefinitions: "Delete Selected Definitions?",
```

Comment:
I know our casing is inconsistent - but should this be:  `"Delete selected definitions?"`